docs/guide/content_rendering/embed_and_list_content/render_images.md

```diff
@@ -67,3 +67,50 @@ To use it, select the variation when rendering the image:
     }
 }) }}
 ```
+
+## Use focal point
+
+In the [image editor](../../image_editor.md) you can define a focal point for an image.
+The focal point doesn't have an instant effect when you use the default templates.
+However, you can use it to select the part of the image the view focuses on when the image is cropped.
+
+The following example shows how to use an image contained in an Image Field as a focussed background.
+
+!!! note
+
+    This implementation is only an example and depends on the JavaScript framework you are using.
+
+First, in the main template, render the Image Field with a custom template:
+
+``` html+twig
+{{ ez_render_field(content, 'image', {
+    'template': 'fields/image.html.twig'
+}) }}
+```
+
+Then, create the custom Field template in `template/fields/image.html.twig`,
```

Comment:
is the path correct? Should it be template/fields... or templates/fields...?

Reply:
```suggestion
Then, create the custom Field template in `templates/fields/image.html.twig`,
```